lib/conf.d/bases_structure.xml

```diff
@@ -123,15 +123,15 @@
             <type>DATETIME</type>
             <null></null>
             <extra></extra>
-            <default></default>
+            <default>0000-00-00 00:00:00</default>
```

Comment:
no invalid dates in strict mode, null is ok if col. is nullable.
https://www.percona.com/blog/2016/10/18/upgrading-to-mysql-5-7-beware-of-the-new-strict-mode/